### PR TITLE
Fix sentry errors

### PR DESCRIPTION
**Pull request by Lyczeq**
**Description**

Changes proposed in this pull request:

-Fixed sentry errors

**Related issue(s)**

### CODE REVIEW

File: core-ui/src/shared/utils/helpers.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ const capitalize = str => {
 };
 
 const splitName = name => {
-  const nameArray = name.match(/[A-Z][a-z]+/g);
+  const nameArray = name?.match(/[A-Z][a-z]+/g);
```
Comment (mrCherry97) [inline core-ui/src/shared/utils/helpers.js:98]: When the name will be undefined there can be a little problem

**Review by mrCherry97 — CHANGES_REQUESTED** [review]

File: core-ui/src/shared/utils/helpers.js (hunk 1 above)
Comment (mrCherry97) [reply core-ui/src/shared/utils/helpers.js:98]: 99 line, not 98 :D

**Review by mrCherry97 — APPROVED** [review]

**Comment by Wawrzyn321** [comment]
bruh /test pre-busola-integration-test-k3s-cluster

**Pull request merged**